src/Kunstmaan/TranslatorBundle/Service/Translator/ResourceCacher.php

```diff
@@ -70,30 +67,41 @@ public function getCacheFileLocation()
 
     /**
      * Remove all cached files (translations/resources)
-     * @return void
+     *
+     * @return bool
      */
     public function flushCache()
     {
         $finder = new Finder();
+        $finder->files()->in('dsdf')->name('*.php');
```

Comment:
I assume this line has to be restored to `$finder->files()->in($this->cacheDir)->name('*.php')`